pipeline/eval/eval.py

```diff
@@ -158,11 +164,11 @@ def main(args_list: Optional[list[str]] = None) -> None:
     elif args.model_variant == "gpu":
         if not args.workspace:
             raise Exception("The workspace size was not provided")
-        if not args.gpus:
-            raise Exception("The number of GPUs was not provided")
+        # if not args.gpus:
```

Comment:
Uncomment or remove?

Reply:
Thanks, that's old code. I'll remove it.